corehq/ex-submodules/phonelog/models.py

```diff
@@ -19,6 +19,9 @@ class DeviceReportEntry(models.Model):
 
     class Meta:
         unique_together = [('xform_id', 'i')]
+        index_together = [
+            ("date", "domain"),
```

Comment:
Shouldn't this be `('domain', 'date')`? Unless you know otherwise, my understanding is that this creates a b-tree where (domain, date) is the key, which means pretty much the same thing as an emit in couch—it means "sort by domain and then by date" which lets you say "for domain foo, give me everything in this date range".

The way you have it ordered, in the understanding I just gave, means you can run queries like "Given a certain exact datetime, give me all the entries sorted by domain"—which is a pretty useless query especially since datetimes aren't discreet the way domains are, but also just since that's not the query we're trying to run.

Do I have this backwards?